### PR TITLE
UIORGS-2 Restructure details pane in organizations

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
To update the view vendor details screen into a view organization details screen, to clean up some of the text, and to suppress vendor-specific accordions when they are not needed.
https://issues.folio.org/browse/UIORGS-2

## Approach
I tried to apply only related changes, to provide less refactoring changes for clearer diff.

#### TODOS and Open Questions
- [ ] Add some tests.

## Learning
* we have postcss-import to support re-use of css https://github.com/postcss/postcss-import;
* css3 prop `overflow-wrap: break-word;` works to wrap long values in `KeyValue` component https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap;

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Interface/InterfaceView.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-/* ContactPeopleView.css */
+@import "../Utils/wrapValue.css";
```
Comment (NikitaSedyx) [inline src/Interface/InterfaceView.css]: css from utils?

Comment (aliaksei-chumakou) [reply src/Interface/InterfaceView.css]: any suggestions?)

Comment (NikitaSedyx) [reply src/Interface/InterfaceView.css]: common folder :)

Comment (aliaksei-chumakou) [reply src/Interface/InterfaceView.css]: so `src/common/styles/wrapValue.css` is fine?

File: test/bigtest/interactors/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { default as OrganizationsListInteractor } from './organizationsList';
```
Comment (NikitaSedyx) [inline test/bigtest/interactors/index.js]: :+1:

**Comment by aliaksei-chumakou** [comment]
tests coverage will be improved later, it's coordinated with product owner

**Pull request merged**